### PR TITLE
connectedvmware 1.1.0

### PR DESCRIPTION
**Pull request by nascarsayan**
Changes:
* Prompt credentials if not provided for Guest Agent.

---

### Related command

`az connectedvmware`

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by subbartt — APPROVED** [review]

File: src/connectedvmware/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.0.2
```
Comment (AllyW) [inline src/connectedvmware/HISTORY.rst:5]: Feature added should mark version to increase in minor part, which is 1.1.0

Comment (Moazzem-Hossain-pixel) [reply src/connectedvmware/HISTORY.rst:5]: Request your favor to solve this issue if possible!

**Review by Moazzem-Hossain-pixel — APPROVED** [review]
Thanks

**Review by Moazzem-Hossain-pixel — CHANGES_REQUESTED** [review]
Please do as well.

**Comment by kairu-ms** [comment]
@nascarsayan Please fix the version issue. Thanks

File: src/connectedvmware/HISTORY.rst (hunk 1 above)
Comment (nascarsayan) [reply src/connectedvmware/HISTORY.rst]: Sorry for the delay. updated.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**